src/tools/kata-ctl/src/check.rs

```diff
@@ -6,32 +6,22 @@
 // Contains checks that are not architecture-specific
 
 use anyhow::{anyhow, Result};
-// See: https://github.com/kata-containers/kata-containers/issues/5438
-#[cfg(any(
-    target_arch = "aarch64",
-    target_arch = "powerpc64le",
-    target_arch = "x86_64"
-))]
 use reqwest::header::{CONTENT_TYPE, USER_AGENT};
 use serde_json::Value;
-#[cfg(any(
-    target_arch = "aarch64",
-    target_arch = "powerpc64le",
-    target_arch = "x86_64"
-))]
 use std::collections::HashMap;
-use std::fs;
 
 const KATA_GITHUB_URL: &str =
     "https://api.github.com/repos/kata-containers/kata-containers/releases/latest";
 
+#[cfg(test)]
 fn get_cpu_info(cpu_info_file: &str) -> Result<String> {
-    let contents = fs::read_to_string(cpu_info_file)?;
+    let contents = std::fs::read_to_string(cpu_info_file)?;
     Ok(contents)
 }
 
 // get_single_cpu_info returns the contents of the first cpu from
 // the specified cpuinfo file by parsing based on a specified delimiter
+#[cfg(test)]
```

Comment:
@hbrueckner the same situation as before, this function is not a module test

Reply:
dito.